### PR TITLE
Added badges

### PR DESCRIPTION
**Pull request by theofidry**
https://www.codacy.com/app/theofidry/bytes-js/issues

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Bytes utility
 
+[![npm version](https://badge.fury.io/js/bytes.svg)](http://badge.fury.io/js/bytes)
+[![Dependency Status](https://www.versioneye.com/nodejs/bytes/2.1.0/badge.svg)](https://www.versioneye.com/nodejs/bytes/2.1.0)
```
Comment (dougwilson) [inline Readme.md]: We have no dependencies. I don't see the purpose of having the dependency badge?

File: Readme.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Bytes utility
 
+[![npm version](https://badge.fury.io/js/bytes.svg)](http://badge.fury.io/js/bytes)
```
Comment (dougwilson) [inline Readme.md]: Can this be a shields.io badge?

Comment (theofidry) [inline Readme.md]: Done

**Comment by theofidry** [comment]
Sorry I didn't see the first comment, dependency badge remvoed.

**Comment by dougwilson** [comment]
Thanks, again!

**Pull request closed without merging**